src/likelihoods/bernoulli.jl

```diff
@@ -18,4 +18,4 @@ BernoulliLikelihood(l=logistic) = BernoulliLikelihood(link(l))
 
 (l::BernoulliLikelihood)(f::Real) = Bernoulli(l.invlink(f))
 
-(l::BernoulliLikelihood)(fs::AbstractVector{<:Real}) = Product(Bernoulli.(l.invlink.(fs)))
+(l::BernoulliLikelihood)(fs::AbstractVector{<:Real}) = Product(l.(fs))
```

Comment:
Could just use `map` now ans avoid the broadcasting machinery since broadcast fusion is not used anymore:

```suggestion
(l::BernoulliLikelihood)(fs::AbstractVector{<:Real}) = Product(map(l, fs))
```